clients/go/main.go

```diff
@@ -127,7 +140,7 @@ func uploadCreationProcedure(file *os.File) (string, error) {
 
 	req, err := http.NewRequestWithContext(ctx, "POST", endpoint, file)
 	req.Header.Set("Upload-Draft-Interop-Version", InteropVersion)
-	req.Header.Set("Upload-Incomplete", "?0")
+	req.Header.Set("Upload-Complete", "?0")
```

Comment:
This should be Upload-Complete: ?1 because the client tries to transmit the entire file in the request.